src/include/network/lan/bridge.rb

```diff
@@ -60,6 +87,21 @@ def configure_as_bridge_port(device)
 
       NetworkInterfaces.Commit
       NetworkInterfaces.Add
+
+      NetworkInterfaces.Current = selected_interface
+    end
+
+    # Asks the user about adapt the current bridge port config in case that it
+    # is configured in the old way (BOOTPROTO == static) & (IPADDR == 0.0.0.0).
+    def fix_bridge_port_config?(ports)
```

Comment:
well it better describes current use case, but i don't see any benefit in having such one purpose method. So do you have any other pros? ;-)